src/state/hooks.ts

```diff
@@ -89,48 +89,15 @@ export const useFarmFromTokenSymbol = (tokenSymbol: string, preferredQuoteTokens
   return filteredFarm
 }
 
+// Return the base token price for a farm, from a given pid
 export const useBusdPriceFromPid = (pid: number): BigNumber => {
   const farm = useFarmFromPid(pid)
-  const bnbPriceBusd = usePriceBnbBusd()
```

Comment:
This function is now effectively a state selector. The logic is being done as part of the farms fetch.